### DISCUSSION TITLE
How to avoid Errors while using polling

### DISCUSSION THREAD

**Discussion by YU-TAYU** [livewire/livewire#3546] [1. Help]
Hello, I would like to know about error avoidance during polling.

I use polling.

When deploying code with the variable $hoge added, if the screen is displayed without reloading, the function will be executed by polling and an "Undefined array key “hoge”" error will be displayed.
Because the deployment is done while the user is using it
I'm thinking about how to automatically reload the screen when changes are made.

I want to avoid the way to detect the event when this change is made or the error itself.

I would like to know if there is a way.

Sample.php
```
class Sample extends Component
{

    public $hoge; // add and deploy

    public function mount()
    {
       $this->hoge = "World"; // add and deploy
    }

    /**
     * view
     */
    public function render()
    {
        return view('livewire.sample');
    }
}
```

sample.blade.php
```
<div wire:poll.1000ms>
   <p>{{ now() }}</p>
   <p>{{ $hoge }}</p> {{-- add and deploy --}}
</div>
```

**Comment by joshhanley** [comment]
@YU-TAYU have a look at PR #2977 it's something that is being looked into 🙂
